app/models/exchange_rates/exchange_rate.rb

```diff
@@ -0,0 +1,37 @@
+module ExchangeRates
+  class ExchangeRate
```

Comment:
why not use the ExchangeRateCurrencyRate class?

Reply:
reason for not using the other file is it this one has different attributes and didnt think worth adding them to the save model and the DB?